classes/author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_accessor :first_name, :last_name, :id, :items
+
+  def initialize(first_name, last_name, id = Random.rand(1..1000))
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
- [ ] Please know that you need to specify the correct 1-to-many-relationship here, this is an example of how it should be: 
```ruby
    def add_item(item)
      @items.push(item)
      item.genre = self
    end
```